[2025-04-07T17:40:43.945+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-07T17:40:43.967+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tmdb_pipeline_streamlit.load_to_postgres manual__2025-04-07T17:37:50.690898+00:00 [queued]>
[2025-04-07T17:40:43.976+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tmdb_pipeline_streamlit.load_to_postgres manual__2025-04-07T17:37:50.690898+00:00 [queued]>
[2025-04-07T17:40:43.977+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-04-07T17:40:43.990+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_to_postgres> on 2025-04-07 17:37:50.690898+00:00
[2025-04-07T17:40:43.998+0000] {standard_task_runner.py:63} INFO - Started process 293 to run task
[2025-04-07T17:40:44.001+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'tmdb_pipeline_streamlit', 'load_to_postgres', 'manual__2025-04-07T17:37:50.690898+00:00', '--job-id', '541', '--raw', '--subdir', 'DAGS_FOLDER/tmdb_pipeline.py', '--cfg-path', '/tmp/tmp5sdznhpc']
[2025-04-07T17:40:44.005+0000] {standard_task_runner.py:91} INFO - Job 541: Subtask load_to_postgres
[2025-04-07T17:40:44.056+0000] {task_command.py:426} INFO - Running <TaskInstance: tmdb_pipeline_streamlit.load_to_postgres manual__2025-04-07T17:37:50.690898+00:00 [running]> on host fcc84ee277a3
[2025-04-07T17:40:44.156+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tmdb_pipeline_streamlit' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-04-07T17:37:50.690898+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-07T17:37:50.690898+00:00'
[2025-04-07T17:40:44.157+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-07T17:40:44.185+0000] {tmdb_pipeline.py:1912} ERROR - No se encontró el archivo de datos procesados: None
[2025-04-07T17:40:44.200+0000] {tmdb_pipeline.py:1922} INFO - Leyendo 20 registros procesados para cargar en PostgreSQL
[2025-04-07T17:40:44.207+0000] {tmdb_pipeline.py:1495} INFO - Tablas creadas o verificadas correctamente en PostgreSQL
[2025-04-07T17:40:44.210+0000] {tmdb_pipeline.py:1938} INFO - Conexión establecida con PostgreSQL
[2025-04-07T17:40:44.231+0000] {tmdb_pipeline.py:2090} INFO - Total de películas en la base de datos: 62
[2025-04-07T17:40:44.232+0000] {tmdb_pipeline.py:2091} INFO - Insertadas 0 nuevas películas en esta ejecución
[2025-04-07T17:40:44.232+0000] {tmdb_pipeline.py:2097} INFO - Datos cargados correctamente en PostgreSQL
[2025-04-07T17:40:44.232+0000] {python.py:237} INFO - Done. Returned value was: 20
[2025-04-07T17:40:44.233+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-07T17:40:44.253+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=tmdb_pipeline_streamlit, task_id=load_to_postgres, execution_date=20250407T173750, start_date=20250407T174043, end_date=20250407T174044
[2025-04-07T17:40:44.309+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-07T17:40:44.336+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-07T17:40:44.338+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
